.github/dependabot.yml

```diff
@@ -11,3 +11,24 @@ updates:
         update-types:
         - "version-update:semver-major"
         - "version-update:semver-minor"
+
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: daily
+    commit-message:
+      prefix: "chore"
+      
+  - package-ecosystem: docker
+    directory: /
+    schedule:
+      interval: daily
+    commit-message:
+      prefix: "chore"
+      
+  - package-ecosystem: gomod
+    directory: /tools
+    schedule:
+      interval: daily
```

Comment:
For the other `gomod` we have weekly. It's fine to have daily here but if it becomes too noisy, we can change this to weekly later.